buildconfig/pygame-stubs/mixer.pyi

```diff
@@ -81,5 +86,5 @@ class Channel:
     def get_busy(self) -> bool: ...
     def get_sound(self) -> Sound: ...
     def get_queue(self) -> Sound: ...
-    def set_endevent(self, type: Optional[Union[int, Event]] = None) -> None: ...
+    def set_endevent(self, type: Union[int, Event] = 0) -> None: ...
```

Comment:
Same concern here as with `mask.connected_component`. Is a default arg of zero the same as not passing an arg at all?

https://www.pygame.org/docs/ref/mixer.html#pygame.mixer.Channel.set_endevent

Reply:
In this case, I had gone through the code, and can confirm that passing 0 indeed does the same thing as not passing an arg at all